cmd/utils.go

```diff
@@ -636,6 +636,51 @@ func CopyDir(log *LoggingConfig, fromDir string, toDir string) error {
 	return nil
 }
 
+// CopyAllContents copies the files and folders from one place to another, without the parent folder
+func CopyAllContents(log *LoggingConfig, fromDir, toDir string) error {
+	var files []string
+
+	// Get the filepaths within the given folder
+	err := filepath.Walk(fromDir, func(path string, info os.FileInfo, err error) error {
```

Comment:
The `filepath.Walk` method will traverse the entire directory tree, including subdirectories. I'm not sure you want this, because below you use `CopyDir` which copies all the contents of a directory. You probably only need to list the top level files and dirs from this method. I suggest looking at `ioutil.ReadDir` - this will already give you a FileInfo so you don't need to call `os.Stat` later.

Alternatively, you could avoid writing this `CopyAllContents` function altogether. Use `CopyDir` to copy the `stackDir` into the `sandbox.ProjectDir`, then reset `sandbox.ProjectDir` to be one level lower (aka the new stackDir)

Reply:
I've chosen to go with an alternative method :) Realised I was being a bit silly and overcomplicated it; the new utils.go function has been removed and instead the `sandbox.ProjectDir` now goes one level deeper into the `starter` folder before running `appsody stack package`

Reply:
Yep I like this much better! Look at it as good coding exercise ;-)
Also great job commenting the code for this exception! Thanks!